server/controllers/content.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable sort-keys */
+/* eslint-disable consistent-return */
+const HttpStatus = require('http-status-codes');
+const Content = require("../models/Content");
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND } = HttpStatus;
+
+// Create and Save a new Content
+exports.create = async(req, res) => {
+    try {
+        const { title, resourceType, tags, url, subjectId } = req.body;
+        // Validate request
+        if (!title) {
+            return res.status(BAD_REQUEST)
+                .send({
+                    status: "fail",
+                    data: { "title": "Content title is required" },
+                });
+        }
+
+        // Create a Content
+        const content = new Content({
+            title,
+            resourceType,
+            tags,
+            url,
+            subjectId,
+        });
+
+        // Save Content in the database
+        const data = await content.save();
+        res.status(CREATED)
+            .send({
+                status: "success",
+                data: { "content": data },
+            });
+    } catch (err) {
+        res.status(INTERNAL_SERVER_ERROR)
+            .send({
+                status: "error",
+                message: HttpStatus.getStatusText(INTERNAL_SERVER_ERROR) ||
+                    "Some error occurred while creating the Content.",
```

Comment:
just ` message: HttpStatus.getStatusText(INTERNAL_SERVER_ERROR)` fix this in your files